includes/wikia/ArticlesUsingMediaQuery.class.php

```diff
@@ -68,7 +68,19 @@ public function getArticleList($purgeCache = false) {
 	}
 
 	public function getMemcKey() {
-		return wfMemcKey( 'ArticlesUsingMediaQuery', 'v3', md5( $this->fileTitle->getDBKey() ) );
+		global $wgScriptPath;
+
+		$params = [
+			'ArticlesUsingMediaQuery',
+			'v3',
+			md5( $this->fileTitle->getDBKey() )
+		];
+
+		if ( !empty( $wgScriptPath ) ) {
+			array_push( $params, $wgScriptPath );
```

Comment:
For adding one variable to an array `$keyParams[] = $wgScriptPath;` is preferred (including in PHP docs).